networks/_modules/ignite/bootstrap.tf

```diff
@@ -88,6 +88,7 @@ resource "local_file" "genesis-file" {
       "eip155Block": 0,
       "eip150Hash": "0x0000000000000000000000000000000000000000000000000000000000000000",
       "eip158Block": 0,
+      "qip714Block": 20,
```

Comment:
If the value `20` is used in the test, please provide comments to reflect the same.

Reply:
this is done.